recipes/rake-nltk/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "rake_nltk" %}
+{% set version = "1.0.3" %}
+{% set sha256 = "67759e32ef9be82bbfc78efdb92db56fc43bc7c98053dcd415f6ddf576b4ea62" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - nltk
+  run:
+    - python
+    - nltk
+
+test:
+  imports:
+    - rake_nltk
+
+about:
+  home: https://github.com/csurfer/rake-nltk
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'RAKE short for Rapid Automatic Keyword Extraction algorithm'
+
+  description: |
+    RAKE short for Rapid Automatic Keyword Extraction algorithm, is a domain independent
+    keyword extraction algorithm which tries to determine key phrases in a body of text
+    by analyzing the frequency of word appearance and its co-occurance with other words
+    in the text.
+  doc_url: https://pypi.org/project/rake-nltk/#files
```

Comment:
This is not a good "doc_url". It is okay to leave it blank if no docs are available.

Reply:
I couldn't find a good doc_url so I deleted this line from the meta.yaml file.